### DISCUSSION TITLE
Different types of channel name

### DISCUSSION THREAD

**Discussion by lilyyff** [ROBelgium/MSNoise#316] [Q&A] [answered]
Hi! 
  I have trouble recognizing the data. Channel names of my data are HH1, HH2, HHZ while some others are BHE, BHN, BHZ. It seems that it can't recognize that HH1, HH2 are also horizontal components when compute-cc, so there is no horizontal component cross-correlation and autocorrelation result.
  Is there any parameter that can be help to recognize different types of channel names? or do I need to manually change them to HHE, HHN...
  Thanks for your reply :)

**Comment by ThomasLecocq** [comment, marked as answer]
You have to mention the pairs in the horizontal components: ZZ,12,Z1,E1 etc

**Reply by lilyyff** [reply]
Thank you! I get it now!!!
